tests/test_store.py

```diff
@@ -214,6 +220,34 @@ def test_view_query(
         # tests that the query was valid and successful, that a view subset has actually been retrived
         assert synapse_store_special_scope.storageFileviewTable.empty is False
 
+    @pytest.mark.parametrize(
+        "asset_view,columns,expectation",
+        [
+            ("syn62339865", ["path"], pytest.raises(ValueError)),
+            ("syn62340177", ["id"], pytest.raises(ValueError)),
+            ("syn23643253", ["id", "path"], does_not_raise()),
+        ],
+    )
+    def test_view_query_exception(
```

Comment:
I am seeing both `test_view_query` and this test. Could you change the both of their name to `test_query_fileview` and test_query_fileview_exception`? I think that will make the test more searchable